### PR TITLE
Update zh trans configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by jiazxjason**
Update zh trans configure-upgrade-etcd.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -404,6 +417,18 @@ etcd 支持从 [major.minor](http://semver.org/) 或其他不同 patch 版本的
...
 虽然已调度的 pod 可能继续运行，但新的 pod 无法调度。在这种情况下，恢复 etcd 集群并可能需要重新配置 Kubernetes API 服务器以修复问题。
 
+{{< note >}}
+如果集群中正在运行任何 API 服务器，则不应尝试还原 etcd 的实例。相反，请按照以下步骤还原 etcd：
+
+- 停止 *所有* kube-apiserver 实例
+- 在所有 etcd 实例中恢复状态
+- 重启所有 kube-apiserver 实例
+
+我们还建议任何组件（例如 kube-scheduler，kube-controller-manager，kubelet），以确保它们不会
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md:427]:
```suggestion
我们还建议重启所有组件（例如 kube-scheduler、kube-controller-manager、kubelet），以确保它们不会
```

**Comment by jiazxjason** [comment]
@tengqm

**Pull request merged**